include/aspect/material_model/interface.h

```diff
@@ -203,6 +203,9 @@ namespace aspect
         entropy_derivative_pressure    = 128,
         entropy_derivative_temperature = 256,
         reaction_terms                 = 512,
+        reaction_rates                 = 1024,
+        force_terms                    = 2048,
```

Comment:
I am not so sure about making this one separate from `additional_outputs`. Structurally it is a subset of `additional_outputs`. I understand we may want to ask for it separately, but in the current form there is a problem: Someone could ask for all `additional_outputs` and then assume all additional outputs will be computed, while `force_terms` will not be computed if not requested. Is there a reason we need this as a separate entry?